tests/test_scxml.py

```diff
@@ -2,7 +2,7 @@
 from typing import Optional, Dict, List
 import json
 import pytest
-from xstate.scxml import scxml_to_machine
+from tests.scxml import scxml_to_machine
```

Comment:
We might want `scxml_to_machine` to be public API

Reply:
Ah yes why not. I'll move it back tomorrow